lib/git-whistles/jira.rb

```diff
@@ -0,0 +1,54 @@
+module Git
+  module Whistles
+    module Jira
+      def get_config
+        username = `git config jira.username`.strip
+        if username.empty?
+          puts Term::ANSIColor.yellow %Q{
```

Comment:
The indentation looks a bit from before this being moved. Do you mind keeping it the same please? 

```
puts Term::ANSIColor.yellow %Q{
  Your branch appears to have a issue ID,
  but I don't know your JIRA username!
  Please set it with:
    $ git config [--global] jira.username <username>
}
```